tests/test_deepgrow_transforms.py

```diff
@@ -329,6 +329,39 @@
 
 RESTORE_LABEL_TEST_CASE_2 = [{"keys": ["pred"], "ref_image": "image", "mode": "nearest"}, DATA_11, RESULT]
 
+RESTORE_LABEL_TEST_CASE_3_RESULT = np.zeros((10, 20, 20))
+for layer in range(5):
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 0:10, 0:10] = 1
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 0:10, 10:20] = 2
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 10:20, 0:10] = 3
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 10:20, 10:20] = 4
+
+for layer in range(5, 10):
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 0:10, 0:10] = 5
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 0:10, 10:20] = 6
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 10:20, 0:10] = 7
+    RESTORE_LABEL_TEST_CASE_3_RESULT[layer, 10:20, 10:20] = 8
+
+RESTORE_LABEL_TEST_CASE_3 = [
+    {"keys": ["pred"], "ref_image": "image", "mode": "nearest", "restore_cropping": False},
+    DATA_11,
+    RESTORE_LABEL_TEST_CASE_3_RESULT,
+]
+
+RESTORE_LABEL_TEST_CASE_4 = [
+    {
+        "keys": ["pred"],
+        "ref_image": "image",
+        "mode": "nearest",
+        "restore_resizing": False,
+        "restore_cropping": False,
+        "restore_spacing": False,
+        "restore_slicing": False,
```

Comment:
We have this test for when all the flags are false and the previous one where only one flag is false, I think we need the other three tests for when one flag at a time is false. This just ensures any future changes doesn't impact behaviour, the tests can be simple like this one if that's easier.